src/shared/src/components/LandmarkShortcuts.tsx

```diff
@@ -0,0 +1,65 @@
+import { makeStyles } from "@material-ui/core";
+import React from "react";
+import cn from 'classnames';
+
+export interface ILandmarkShortcutsProps {
+  navToMainText: string;
+}
+
+const useStyles = makeStyles({
+  button: {
+    textDecoration: 'underline',
+    textAlign: 'left',
+    width: 'fit-content',
```

Comment:
maybe some padding would be nice?
also not sure if 'fit-content' does much here.

Reply:
Agreed, now has same padding as altinn.no design.